src/style.css

```diff
@@ -1 +1,39 @@
-/* add styles here */
+body {
+  margin: 0;
+}
```

Comment:
You've used a tag name for styling the body. As per the checklist, it is recommended to avoid using tag names for styling (except `html` and `body`). However, in this case, it's acceptable as the `body` tag is one of the exceptions. Just be careful not to use other tag names for styling.